estate/models/estate_property_offer.py

```diff
@@ -10,16 +10,28 @@ class EstatePropertyOffer(models.Model):
     status = fields.Selection(
         copy=False,
         selection=[
-            ('accepted', 'Accepted'),
-            ('refused', 'Refused'),
+            ("accepted", "Accepted"),
+            ("refused", "Refused"),
         ],
     )
     partner_id = fields.Many2one(
-        comodel_name='res.partner',
-        string='Partner',
+        comodel_name="res.partner",
+        string="Partner",
         required=True,
     )
     property_id = fields.Many2one(
-        comodel_name='estate.property',
+        comodel_name="estate.property",
         required=True,
     )
+    create_date = fields.Date(default=fields.Date.today())
```

Comment:
The date of fields.Date.today() will always be equal to the date the server was started if you don't use a lambda function.

```suggestion
    create_date = fields.Date(default=lambda self: fields.Date.today())
```